.github/workflows/linting.yml

```diff
@@ -0,0 +1,11 @@
+name: Linting
```

Comment:
Could you mention where you get this settings from? I believe all apps share the same lint/scan config, right?